neurom/features/neuronfunc.py

```diff
@@ -230,14 +229,18 @@ def _count_crossings(neurite, radius):
         for start, end in iter_segments(neurite):
             start_dist2, end_dist2 = (morphmath.point_dist2(center, start),
                                       morphmath.point_dist2(center, end))
-
             count += int(start_dist2 <= r2 <= end_dist2 or
                          end_dist2 <= r2 <= start_dist2)
 
         return count
 
+    if hasattr(neurites, 'neurites') or hasattr(neurites, 'points'):
```

Comment:
This block does not make much sense. Why `neuron_population` function can't be reused. Why cast everything to `list`? There was a nice `iter_neurites` below which now is dropped. Why?

Reply:
this is the simplest I could do to accept a list of sections. Maria gets a list of section (mabye some in the apical tuft), and wants to compute sholl. The iter_neurites is still under the if l 237, which works if you give a neuron or a neurites, but not a list of sections. I had to convert to list fr some reason I don't recall

Reply:
maybe there is a cleaner way to do this, but I couldn't figure it out. Maybe with v2?

Reply:
Lets try this function. Can it be used instead? Also I've updated its docstring more.
```python
def sholl_crossings(neurites, center, radii):
    """Calculate crossings of neurites.

    This function can also be used with a list aa neurites, as follow:

        secs = (sec for sec in nm.iter_sections(neuron) if complex_filter(sec))
        sholl = nm.features.neuronfunc.sholl_crossings(secs,
                                                       center=neuron.soma.center,
                                                       radii=np.arange(0, 1000, 100))

    Args:
        neurites(list): morphology on which to perform Sholl analysis, or list of neurites
        center(Point): center point
        radii(iterable of floats): radii for which crossings will be counted

    Returns:
        Array of same length as radii, with a count of the number of crossings
        for the respective radius
    """
    def _count_crossings(neurite, radius):
        """Used to count_crossings of segments in neurite with radius."""
        r2 = radius ** 2
        count = 0
        for start, end in iter_segments(neurite):
            start_dist2, end_dist2 = (morphmath.point_dist2(center, start),
                                      morphmath.point_dist2(center, end))
            count += int(start_dist2 <= r2 <= end_dist2 or
                         end_dist2 <= r2 <= start_dist2)

        return count

    return np.array([sum(_count_crossings(neurite, r)
                         for neurite in iter_neurites(neurites))
                     for r in radii])
```

Reply:
> this is the simplest I could do to accept a list of sections. Maria gets a list of section (mabye some in the apical tuft), and wants to compute sholl. The iter_neurites is still under the if l 237, which works if you give a neuron or a neurites, but not a list of sections. I had to convert to list fr some reason I don't recall

Can you add a test then for this case of list of sections?